Dockerfile

```diff
@@ -37,4 +63,19 @@ ENV FLASK_ENV=development
 
 WORKDIR /app/todo_files
 ENTRYPOINT [ "poetry", "run", "flask", "run" ]
-CMD [ "--host", "0.0.0.0" ]  
\ No newline at end of file
+CMD [ "--host", "0.0.0.0" ]
+
+
+FROM base as test
+# Configure for production
+ENV FLASK_ENV=test
+
+RUN pip3 install pytest
```

Comment:
Why are we installing all of these with pip instead of with poetry?

Better practice would be to specify our test dependencies in our package manager along with dev-dependencies, e.g. adding pytest:
`poetry add pytest -D `

We can then specify `poetry install --no-dev` for our production container, and just `poetry install` for the others, and we don't need to update this dockerfile every time our test dependencies change